compensated-rails/spec/dummy/config/application.rb

```diff
@@ -8,7 +8,7 @@
 module Dummy
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
+    config.load_defaults Rails.version.slice(0,3)
```

Comment:
This allows us to support Rails 5.X~6.X; it may not be _the right_ way to do it, but I was doing something quick and dirty to get CI passing. Maybe we should look at how other Rails engines test across multiple Rails?

Reply:
A bit confused on how does it allow us to support Rails 5.X ~ 6.X?
This is related to changing autoloading mode?
Saw this from https://guides.rubyonrails.org/configuring.html#rails-general-configuration
> config.autoloader sets the autoloading mode. This option defaults to :zeitwerk if 6.0 is specified in config.load_defaults.

Reply:
Without this change when we tried to include compensated-rails in a Rails 5 project, it would crash out because the value was hardcoded to `6.0` , which is not a valid option in Rails 5.

I'm not sure the _right_ thin to do here, but this at least gets stuff passing.